acdc-ws/conf/routes

```diff
@@ -1,3 +1,4 @@
 GET     /__status                   controllers.StatusController.status()
+GET     /__metrics                  controllers.StatusController.metrics()
```

Comment:
I was hoping this can be configurable

Reply:
I can look into this.  I am thinking about moving this to ApiRouter, where I can read the config file.